libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -1193,7 +1193,7 @@ export function updateParameterWithValues(
                     isUnknown: true,
                   };
 
-                  parameters.push(transformInputParameter(restInputParameter, propertyValue, /* invisible */ false));
+                  parameters.push(transformInputParameter(restInputParameter, propertyValue, /* invisible */ true));
```

Comment:
Any manifest defined parameter should never reach here, there must be somewhere before this code..

Reply:
Updated SchemaProcessorOptions to include internal instead. That function is only called for getInputParametersFromManifest where the issue was happening earlier.